rclcpp/topics/minimal_subscriber/static_wait_set_subscriber.cpp

```diff
@@ -35,15 +35,15 @@ class StaticWaitSetSubscriber : public rclcpp::Node
         rclcpp::CallbackGroup::SharedPtr cb_group_waitset = this->create_callback_group(
```

Comment:
I think this `cb_group_waitset` should be kept its lifecycle. What about putting it in the `StaticWaitSetSubscriber::cb_group_waitset_`? Same for the `rclcpp/topics/minimal_subscriber/time_triggered_wait_set_subscriber.cpp`, `rclcpp/topics/minimal_subscriber/wait_set_subscriber.cpp` and `rclcpp/wait_set/src/listener.cpp`.

Reply:
right, example node instance should keep the ownership of CallbackGroup.

Reply:
fixed in https://github.com/ros2/examples/commit/1fc83d440124d76cf23b97f4f4ed5742d61faf95

Reply:
Thanks for the update. Could you tell me why not update the `rclcpp/wait_set/src/listener.cpp`?

Reply:
Ah 😅 that was just an oversight. thanks. i will fix it accordingly.